rtgui/whitebalance.cc

```diff
@@ -543,8 +660,16 @@ void WhiteBalance::optChanged ()
             bool autit = (currMethod.ppLabel == "autitcgreen");
             if (autit) {
                 StudLabel->show();
+                PatchLabel->show();
+                PatchlevelLabel->show();
+                equal->hide();
+                itcwbFrame->set_sensitive(true);
```

Comment:
For greying-out the ITCWB settings in <=5.9 mode, this simple change will work.
```suggestion
                itcwbFrame->set_sensitive(!pp->wb.itcwb_sampling);
```